src/Segmentation/regionprops.jl

```diff
@@ -146,10 +470,10 @@ julia> bw_img = rand([0, 1], 5, 10)
   0  1  0  1  0  0  0  0  4  0
   1  0  0  0  0  2  0  4  0  4
 
- julia> regions = regionprops(label_img, bw_img);
+ julia> measures = regionprops(label_img, bw_img);
 
- julia> for region in regions
-           println(region.area,"\t", region.perimeter)
+ julia> for s in unique(label_img)
+           println(region.area[s],"\t", region.perimeter[s])
```

Comment:
Should this be using `measures` instead of `region`?
```suggestion
           println(measures.area[s],"\t", measures.perimeter[s])
```